grub-core/loader/i386/efi/linux.c

```diff
@@ -283,11 +286,16 @@ grub_cmd_linux (grub_command_t cmd __attribute__ ((unused)),
       goto fail;
     }
 
+  buf = &lh;
   grub_memcpy (linux_cmdline, LINUX_IMAGE, sizeof (LINUX_IMAGE));
   grub_create_loader_cmdline (argc, argv,
                               linux_cmdline + sizeof (LINUX_IMAGE) - 1,
 			      lh.cmdline_size - (sizeof (LINUX_IMAGE) - 1));
-
+  grub_memcpy (linux_cmdline + grub_strlen(linux_cmdline), VERITY_ARG,
+	       sizeof (VERITY_ARG));
+  cmdline_len = grub_strlen(linux_cmdline);
+  grub_memcpy (linux_cmdline + cmdline_len, buf + 0x40, 64);
```

Comment:
Would be nice to have a symbolic names for 0x40 and 64

Also, we should verify the data at that location looks like a hash for the sake of safety and to avoid adding `VERITY_HASH=Use a boot loader...` to the command line when using an unmodified kernel.